### ISSUE TITLE
Too many attributes for a device

### ISSUE THREAD

**Issue by thehands1** [open-dash/open-dash-diy#4]
There are some devices that have more attributes that can fit in a single box causing overlap in the box since it crashes with the bottom text that houses the name.

**Comment by pstuart** [comment]
Yeah, the default template needs work.  Need ideas on what the default tile should look like.  Should it just say, can't find a matching template? or a ?
